about.css

```diff
@@ -95,6 +96,109 @@ section .text-content {
 	text-align: center;
 }
 
+/* Modal */
+
+/* overlay */
+.overlay {
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	/* From https://css.glass */
+	background: rgba(255, 255, 255, 0.17);
+	box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+	backdrop-filter: blur(4.9px);
+	-webkit-backdrop-filter: blur(4.9px);
+}
+
+/* positioning for body to stop scroll when modal is open */
+.fixed {
+	overflow: hidden;
+}
+
+/* Team Member Modal */
+.modal {
+	width: 90%;
+	max-width: 1200px;
+	height: fit-content;
+	max-height: 95vh;
+	margin: 0 auto;
+	padding: 15px;
+	background: rgb(140, 82, 255);
+	background: linear-gradient(
+		80deg,
+		rgba(140, 82, 255, 1) 23%,
+		rgba(196, 124, 214, 1) 84%
+	);
+	border-radius: 30px;
+	position: fixed;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.close-btn {
+	align-self: flex-end;
+	padding: 10px;
+}
+
+.close-btn:hover {
+	cursor: pointer;
+}
+
+.modal-img-box {
+	margin-top: -50px;
+}
+.modal-img {
+	width: 150px;
+	height: 150px;
+	border: 5px white solid;
+	border-radius: 50%;
+	object-fit: cover;
+}
+
+.modal h2 {
+	font-size: var(--font-mobile-lg);
+	margin-top: 10px;
+}
+
+.modal h3 {
```

Comment:
We don't have a font being applied here, it's falling on the browser default. We should probably apply the Inter font to `html` or `body` so that it's the default if no other font is declared.

<img width="306" alt="Screen Shot 2024-04-17 at 11 58 51 AM" src="https://github.com/mothertn/mother-tongues-network-website/assets/704697/243a78fc-23e4-4e9c-901d-ca425b4ab6f5">